Source/SwiftLintFramework/Rules/RuleConfigurations/XCTSpecificMatcherRuleConfiguration.swift

```diff
@@ -0,0 +1,35 @@
+struct XCTSpecificMatcherRuleConfiguration: SeverityBasedRuleConfiguration, Equatable {
+    private(set) var severityConfiguration = SeverityConfiguration(.warning)
+    private(set) var matchers = Set(Matcher.allCases)
+
+    enum Matcher: String, Hashable, CaseIterable {
+        case singleArgument = "single-argument"
+        case doubleArgument = "double-argument"
```

Comment:
What about `one-argument-asserts` and `two-arguments-asserts` like used in the rule?

Reply:
Sounds good.

Reply:
Minor nit: I'll go with `two-argument-asserts` because it represents two-argument asserts and it makes the slugs more similar in formatting (`{one,two}-argument-asserts`).